project/frontend/src/components/ProfileForm.js

```diff
@@ -0,0 +1,69 @@
+import './ProfileForm.css';
+
+import React, { useState } from 'react';
+
+import CuisineAutocomplete from './CuisineAutocomplete.js';
+import ButtonGroup from './ButtonGroup.js';
+
+import Place from '../assets/place.svg';
+import Cuisine from '../assets/cuisine.svg';
+import Distance from '../assets/distance.svg';
+import Experience from '../assets/food_service.svg';
+import Price from '../assets/dollar.svg';
+
+/**
+ * A form used to gather the restaurant preferences of a user for the 'Find My Match'
+ * service.
+ *
+ * @param {!Object<string, number>} currLocation The current latitude and longitude
+ *     coordinates of the user.
+ * @param {string} locationName The formatted address of the current user.
+ */
+function ProfileForm(props) {
+  const cuisineOptions = ['Italian', 'Mexican', 'Indian'];
+  const [cuisine, setCuisine] = useState([]);
+
+  return (
+    <div className='profile-form-container'>
+      <form className='profile-form'>
+        <h4 className='profile-header'>Your profile.</h4>
+        <div className='profile-form-row'>
+          <img src={Place} alt='' />
+          <label>Your Location</label>
+          {/* TODO: change so that if a location isn't inputted then we are going to need to have a fill in optiob. */}
```

Comment:
We could just add a LocationFinder here with sendData={handleLocationData} where handleLocationData sets the state or does whatever we want do do with the location.